### PR TITLE
Helm chart updates

### PR DESCRIPTION
**Pull request by The-Loeki**
* fix dual/inconsistent `slac.bot_name` / `slack.botName`, 'insecure_registry`/`insecureRegistry`
* bump version
* lint 
* set RBAC to default enable (new guidelines, makes sense too)

### CODE REVIEW

**Comment by The-Loeki** [comment]
there you go... Odd one in the `gen-deploy.sh`: `sed -i "" 's/etc'` presumably is supposed to mean `-i without suffix <statement> <file>` but my sed takes it as `-i <statement> <file> <ah-nevermind-i-give-up>`

**Comment by rimusz** [comment]
`sed -i ""` is for Mac, on linux `''"` is not needed, will make an update to check OS version

**Comment by rusenask** [comment]
@rimusz I think gen-deploy is slightly broken as it generates whr sidecars for deployments that shouldn't have them. I am not sure why it happens since it does work correctly when used with helm :/

**Comment by The-Loeki** [comment]
@rusenask well I did think that that `sed 's/false/true` is a bit on the blunt side there ;) that's probably the reason?

File: deployment/deployment-norbac.yaml [hunk 1]
```diff
@@ -129,7 +141,33 @@ spec:
               memory: 128Mi
             requests:
               cpu: 50m
-              memory: 64Mi            
+              memory: 64Mi
+            
+        - name: webhookrelayd
```
Comment (rusenask) [inline deployment/deployment-norbac.yaml]: this shouldn't have been generated

**Comment by rusenask** [comment]
Hmm, not sure whether that's the case, shouldn't helm just completely omit those parts?

File: deployment/scripts/gen-deploy.sh [hunk 2]
```diff
@@ -16,7 +16,7 @@ gen() {
 
 	mkdir -p "$(dirname ${OUTPUT})"
   cp chart/keel/values.yaml "${TMP_VALUES}"/
-  sed -i "" 's/false/true/g' "${TMP_VALUES}/values.yaml"
+  sed -i 's/false/true/g' "${TMP_VALUES}/values.yaml"
```
Comment (The-Loeki) [inline deployment/scripts/gen-deploy.sh]: not if you do this to the values.yaml ;)

Comment (The-Loeki) [reply deployment/scripts/gen-deploy.sh]: perhaps it's better to omit the sed altogether and just enable the values with `--set` in the helm cli directly.

But as I said, the only ones that might work out-of-the-box seems to be `debug`, `podDisruptionBudget` and `service`.

Shall I put those in & see what happens?

Comment (rusenask) [reply deployment/scripts/gen-deploy.sh]: I haven't been using helm for quite some time so not sure how it should work. What do you think @rimusz ? Maybe try generating it and see whether sidecar container still remains there?

**Comment by rusenask** [comment]
merging, fill fix manifests manually

**Pull request merged**